pkg/actuators/machine/actuator_test.go

```diff
@@ -105,32 +137,32 @@ func TestMachineEvents(t *testing.T) {
 		name        string
 		machineName string
 		error       string
-		operation   func(actuator *Actuator, machine *machinev1.Machine)
+		operation   func(actuator *Actuator, machine *machinev1beta1.Machine)
 		event       string
 	}{
 		{
 			name:        "Create machine failed on invalid machine scope",
 			machineName: "test-machine",
-			operation: func(actuator *Actuator, machine *machinev1.Machine) {
+			operation: func(actuator *Actuator, machine *machinev1beta1.Machine) {
 				actuator.Create(nil, machine)
 			},
 			event: "context and machine should not be nil",
 		},
 		{
 			name:        "Create machine failed on missing required label",
 			machineName: "test-machine",
-			operation: func(actuator *Actuator, machine *machinev1.Machine) {
-				machine.Labels[machinev1.MachineClusterIDLabel] = ""
+			operation: func(actuator *Actuator, machine *machinev1beta1.Machine) {
+				machine.Labels[machinev1beta1.MachineClusterIDLabel] = ""
 				actuator.Create(ctx, machine)
 			},
 			event: "missing \"machine.openshift.io/cluster-api-cluster\" label",
 		},
 		/*{
```

Comment:
These test cases are commented out, because running them requires connections to a real nutanix Prism Central endpoint. And when running them out of the Nutanix VPN, they will fail. I didn't removed them but just comment out, since when I run the unit tests locally within the Nutanix VPN, I could uncomment them and run these tests. I will add  a note for this.

Reply:
Removed the comment out code, since we will move/combine this to our integration test suite.